pkg/controller/volume/ephemeral/controller.go

```diff
@@ -279,6 +283,14 @@ func (ec *ephemeralController) handleVolume(pod *v1.Pod, vol v1.Volume) error {
 		Spec: ephemeral.VolumeClaimTemplate.Spec,
 	}
 	_, err = ec.kubeClient.CoreV1().PersistentVolumeClaims(pod.Namespace).Create(context.TODO(), pvc, metav1.CreateOptions{})
+	reason := ""
```

Comment:
Having `reason` be an empty string is going to make querying this metric awkward. I think we probably want a label that distinguishes between success and failure, or even have separate metrics for those.

Reply:
I'm fine with a label `success` that has values `yes` and `no`. Is that okay or are there better names? I am not sure what is used elsewhere.

Reply:
The best practice is to have two metrics one for failures and one for all. Then you can naturally query `failures / total` and understand the percentage. The number of series you produce is identical, so this doesn't produce any more load or anything on the metrics system than what we already have here.

Reply:
I've changed to to count `ephemeral_volume_controller_create_total` and `ephemeral_volume_controller_create_failures`. Is that okay?

Note that my updated tests initially failed because `GetCounterMetricValue` didn't work for a Counter. I'm including the fix for that (= adding the missing Metric interface implementation).

Reply:
Yep, that works!